cypress.json

```diff
@@ -0,0 +1,4 @@
+{
+  "baseUrl": "http://localhost:3000",
```

Comment:
Might want to change this to `https://localhost:4000` if #30 gets approved as is.

Reply:
Changed